actors/miner/src/lib.rs

```diff
@@ -4536,38 +4539,40 @@ fn handle_proving_deadline(
             e.downcast_default(ExitCode::USR_ILLEGAL_STATE, "failed to advance deadline")
         })?;
 
-        // Faults detected by this missed PoSt pay no penalty, but sectors that were already faulty
-        // and remain faulty through this deadline pay the fault fee.
-        let penalty_target = pledge_penalty_for_continued_fault(
-            reward_smoothed,
-            quality_adj_power_smoothed,
-            &result.previously_faulty_power.qa,
-        );
+        if result.previously_faulty_power.qa.is_positive() {
```

Comment:
Has anything changed about our assumptions of previously fault power to make this necessary? If not I would rather err on the side of not messing with any code here.

Reply:
no, but this code has no effect if `!result.previously_faulty_power.qa.is_positive()`, it all ends up in `expected_reward_for_power` where zero QAP always gets us zero

Reply:
I see, this saves us CumSum calculation in the common case which is probably a decent optimization.  Ok and is_positive is simple enough that it can't error.  I'm ok with this.